src/to_string.c

```diff
@@ -101,7 +93,47 @@ size_t number_to_string(char *const dest, uint64_t number) {
     return length;
 }
 
-#define DECIMAL_DIGITS 8
+#define DECIMAL_DIGITS 9
+#define NANO_AVAX_SCALE 1000000000
+
+// Display avax in human readable form
+size_t nano_avax_to_string(char *const dest, uint64_t nano_avax) {
```

Comment:
I'm wondering if this function should be marked `static` or not.

Reply:
This is referenced in apdu_sign.c